modules/security-lti/src/main/java/org/opencastproject/security/lti/LtiLaunchAuthenticationHandler.java

```diff
@@ -315,37 +317,36 @@ public Authentication createAuthentication(HttpServletRequest request, ConsumerA
     userAuthorities.add(new SimpleGrantedAuthority("ROLE_USER"));
     userAuthorities.add(new SimpleGrantedAuthority("ROLE_ANONYMOUS"));
 
-
     // Create/Update the user reference
     if (createJpaUserReference) {
-      JpaOrganization organization = fromOrganization(securityService.getOrganization());
-
-      JpaUserReference jpaUserReference = userReferenceProvider.findUserReference(username, organization.getId());
-
-      Set<JpaRole> jpaRoles = new HashSet<>();
-      for (GrantedAuthority authority : userAuthorities) {
-        jpaRoles.add(new JpaRole(authority.getAuthority(), organization));
-      }
-
-      Date loginDate = new Date();
+      if (attempts.putIfAbsent(username, Boolean.TRUE) != null) {
+        logger.warn("Concurrent access of user {}. Igoring.", username);
```

Comment:
This now becomes expected behavior and is not something we need to warn users about, is it?

```suggestion
        logger.debug("Concurrent access of user {}. Igoring.", username);
```